Userland/Utilities/shot.cpp

```diff
@@ -60,16 +60,14 @@ class SelectableLayover final : public GUI::Widget {
 
     virtual void paint_event(GUI::PaintEvent&) override
     {
-        if (m_region.is_empty()) {
-            GUI::Painter painter(*this);
-            painter.fill_rect(m_window->rect(), m_background_color);
+        GUI::Painter painter(*this);
+        painter.clear_rect(m_window->rect(), m_background_color);
+        painter.fill_rect(m_window->rect(), m_background_color);
```

Comment:
Seems like this just doubles the alpha of the background colour?

Reply:
How does it look now?

Reply:
Code looks alright.
I'll take a look at the result a little later.